### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'comments_before')

### DISCUSSION THREAD

**Discussion by FS-RahulKumarVarlekar** [webpack/webpack#16586] [Show and tell]
we are facing some error while we try to create bundle using webpack node package, can any one have those know the how we can solve problem and you can show problem statement in below image URL.

https://user-images.githubusercontent.com/118047602/209086380-3f9ddf7e-a80c-4a48-b241-1d474f7b025c.png

**Comment by alexander-akait** [comment]
@RahulKumarVarlekar intresing, sounds like a bug in `terser`, what is `terser` version (`npm ls terser`)?
